cwms-data-api/src/main/java/cwms/cda/api/LevelsController.java

```diff
@@ -344,6 +316,12 @@ public void getAll(@NotNull Context ctx) {
                         + "specified), as well as the time zone of any times in the response."
                         + " If this field is not specified, the default time zone of UTC "
                         + "shall be used."),
+                @OpenApiParam(name = START, type = Instant.class, description = "Specifies the start of the time "
```

Comment:
Adding a start and end date for getOne does not make sense to me as it conflicts with the effective date parameter.

Reply:
That's fair, but because there are multiple effective dates there does need to be a way to retrieve them all without it necessarily being a "timeseries" as that's a different concept (seasonal, etc) of location level.

Reply:
I don't see how that's different than the behavior of the getAll endpoint.

Reply:
We'll see how Daniel responds.

Reply:
@DanielTOsborne